src/dataDisplay/personCoreMilestones/gridColumnClassBaseballDiamond.jsx

```diff
@@ -483,7 +481,7 @@ function GridColumnClassBaseballDiamond(props) {
 
     const milestonesClassesDates = [
         ...(attendedClass101Date ? [{ label: '101', date: attendedClass101Date }] : []),
-        ...(signedMembershipAgreementDate ? [{ label: 'Became Member', date: signedMembershipAgreementDate }] : []),
+        ...(congregationDate ? [{ label: 'Became Member', date: congregationDate }] : []),
```

Comment:
Interesting!  We were using `signedMembershipAgreementDate` instead of `congregationDate`.  That was the bug.

Reply:
Yeah, just some requirement confusion.